src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -2,13 +2,14 @@ package cromwell.engine.db
 
 import cromwell.binding.values.WdlValue
 import cromwell.binding.{Call, FullyQualifiedName, WdlJson, WdlSource}
-import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.backend.Backend
-import cromwell.engine.{SymbolStoreKey, SymbolStoreEntry, WorkflowId, WorkflowState}
+import cromwell.engine.{SymbolStoreEntry, WorkflowId, WorkflowState}
 
 import scala.concurrent.Future
 
 object DataAccess {
+  def apply() = new slick.SlickDataAccess()
```

Comment:
Why not just have callers do `SlickDataAccess()` to make clear what's happening?